Framework/Algorithms/src/ConvertUnits.cpp

```diff
@@ -122,15 +155,15 @@ void ConvertUnits::exec() {
     }
   }
 
-  if (inputWS->x(0).size() < 2) {
+  if (correctWS->x(0).size() < 2) {
```

Comment:
Maybe it's not as clear as I thought, I'll put a little comment in basically saying what I said above.